components/life-cycle-extension/src/main/java/com/wso2telco/dep/lifecycleextension/util/Constants.java

```diff
@@ -0,0 +1,26 @@
+package com.wso2telco.dep.lifecycleextension.util;
+
+public class Constants {
+
+    public static final String STATE_CREATED="Created";
+    public static final String ADMIN_USERNAME="admin";
```

Comment:
why do we need this variable?

Reply:
STATE_CREATED="Created" for validate current state when execution APICustomExecutor,
ADMIN_USERNAME="admin" now unuse i will remove this